staging/src/k8s.io/kubectl/pkg/cmd/util/default-container.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+// DefaultContainerDesc defines the description information for the --container command which appears in the following files, such as attach.go, cp.go, exec.go, etc,
+const DefaultContainerDesc = "Container name. If omitted, use the kubectl.kubernetes.io/default-container annotation for selecting the container to be attached or the first container in the pod will be chosen"
```

Comment:
Instead of adding a new file for this what do you think about creating a helper [here](https://github.com/kubernetes/kubernetes/blob/770a9504ea3fe38a75c51f036187b5dea124ebab/staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go#L433) and using it?

Reply:
Thanks, It seems like a better way to do it